tiatoolbox/annotation/storage.py

```diff
@@ -1843,6 +1853,40 @@ def bquery(
             }
         return {key: bounds for key, *bounds in cur.fetchall()}
 
+    @staticmethod
+    def __handle_callable_pquery(unique, select, cur, where):
+        """Handles Callable pquery."""
```

Comment:
No explanation of what "handling" is here. No args or returns etc.

Reply:
@measty and @John-P 
Please can you help with this comment?